jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -334,3 +334,22 @@ def test_correct_output_flux_or_sb_units(cubeviz_helper, spectrum1d_cube_custom_
     # and that calculated moment has the correct units
     mm.calculate_moment()
     assert mm.moment.unit == moment_unit
+
+    uc._obj.show_translator = True
+    uc.flux_or_sb.selected = 'Flux'
+    mm._set_data_units()
+
+    # and make sure this change is propogated
+    output_unit_moment_0 = mm.output_unit_items[0]
+    assert output_unit_moment_0['label'] == 'Flux'
+    assert output_unit_moment_0['unit_str'] == 'Jy'
+
+    # TODO: Failing because of dev version of upstream dependency, figure
+    #  out which one
+    # assert mm.calculate_moment()
+
+    # TODO: This test should pass once continuum subtraction works with
+    #  flux to surface brightness conversion
+    # mm.continuum.selected = 'Surrounding'
+    #
+    # assert mm.calculate_moment()
```

Comment:
These tests will need to be uncommented when we make the unit conversion translator available to users. Otherwise, this PR is ready for a re-review @bmorris3 .